### PR TITLE
Main refactor

### PR DESCRIPTION
**Pull request by mccullya**
Massive cleanup of all the base - all have been tested successfully.

### CODE REVIEW

File: base/cfk-components/confluent-no-auth-no-tls/restproxy.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+kind: KafkaRestProxy
+metadata:
+  name: kafkarestproxy
+spec:
+  replicas: 1
+  dependencies:
+    kafka:
+      bootstrapEndpoint: kafka:9071
+    schemaRegistry:
+      url: https://schemaregistry:8081
```
Comment (sionsmith) [inline base/cfk-components/confluent-no-auth-no-tls/restproxy.yaml:11]: Does this need to be https?

File: base/cfk-components/confluent-no-auth-tls/kustomization.yaml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - secrets
-  - zookeeper.yaml
+  - ../../../resources/application-secrets-rbac
+  - ../cp-version
+  - rest-class.yaml
```
Comment (sionsmith) [inline base/cfk-components/confluent-no-auth-tls/kustomization.yaml:6]: Don't think we need the rest class unless we are using RBAC? not sure though

File: base/cfk-components/cp-version/control-centre.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: platform.confluent.io/v1beta1
+kind: ControlCenter
+metadata:
+  name: controlcenter
+spec:
+  image:
+    application: confluentinc/cp-enterprise-control-center:7.2.0
```
Comment (sionsmith) [inline base/cfk-components/cp-version/control-centre.yaml:7]: 7.2.1 is not available, can update these?

**Review by sionsmith — CHANGES_REQUESTED** [review]
Have a look at what I have put mate, see what you think,

File: base/cfk-components/confluent-no-auth-tls/kustomization.yaml (hunk 2 above)
Comment (mccullya) [reply base/cfk-components/confluent-no-auth-tls/kustomization.yaml:6]: Hmmm... yea, I've always been somewhat confused about that.  I'll test, and remove if not required.

File: base/cfk-components/confluent-no-auth-no-tls/restproxy.yaml (hunk 1 above)
Comment (mccullya) [reply base/cfk-components/confluent-no-auth-no-tls/restproxy.yaml:11]: nice catch

File: base/cfk-components/cp-version/control-centre.yaml (hunk 3 above)
Comment (mccullya) [reply base/cfk-components/cp-version/control-centre.yaml:7]: done

File: base/cfk-components/confluent-no-auth-tls/kustomization.yaml (hunk 2 above)
Comment (mccullya) [reply base/cfk-components/confluent-no-auth-tls/kustomization.yaml]: Right you are.... removed it from the other RBAC example as well :+1:

**Pull request merged**